web/src/main/java/org/mskcc/cbio/oncokb/config/MvcConfigurationPublic.java

```diff
@@ -68,6 +71,22 @@ public Docket publicApi() {
                 "https://www.oncokb.org/terms"
             ))
             .useDefaultResponseMessages(false);
+        
+        // Update swagger host with environment variable
+        String swaggerHost = PropertiesUtils.getProperties("swagger_hostname");
+        if (!StringUtils.isNullOrEmpty(swaggerHost)) {
```

Comment:
I remember it has a method isNotEmpty?

Reply:
The file already had StringUtils imported from a different library: `com.mysql.jdbc`. I changed to use the StringUtils from apache library which has the isNotEmpty method.